### PR TITLE
[ES] content/es/security-chaos-engineering.md

### PR DESCRIPTION
**Pull request by marianogg9**
### Describe your changes
- added `content/es/security-chaos-engineering.md`.

### Related issue number or link (ex: `resolves #issue-number`)
- #1768 

### Checklist before opening this PR (put `x` in the checkboxes)
- [X] This PR does not contain plagiarism
  - don’t copy other people’s work unless you are quoting and contributing it to them.
- [X] I have signed off on all commits 
  - [signing off](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits) (ex: `git commit -s`) is to affirm that commits comply [DCO](https://wiki.linuxfoundation.org/dco). If you are working locally, you could add an alias to your `gitconfig` by running `git config --global alias.ci "commit -s"`.

### CODE REVIEW

File: content/es/security-chaos-engineering.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
+---
+title: Seguridad en Ingeniería del caos
+status: Completed
+category: concepto
```
Comment (ramrodo) [inline content/es/security-chaos-engineering.md:4]:
```suggestion
category: Concepto
```

File: content/es/security-chaos-engineering.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+tags: ["security", "methodology", ""]
+---
+
+## ¿Qué es?
+
+Seguridad en ingeniería del caos, o SCE en Inglés, es una disciplina basada en la [ingeniería del caos](/es/chaos-engineering/).
```
Comment (ramrodo) [inline content/es/security-chaos-engineering.md:10]:
```suggestion
Seguridad en ingeniería del caos, o SCE en Inglés, es una disciplina basada en la [Ingeniería del Caos](/es/chaos-engineering/).
```

File: content/es/security-chaos-engineering.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
+---
+title: Seguridad en Ingeniería del caos
```
Comment (ramrodo) [inline content/es/security-chaos-engineering.md:2]:
```suggestion
title: Seguridad en Ingeniería del Caos
```

File: content/es/security-chaos-engineering.md [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+status: Completed
+category: concepto
+tags: ["security", "methodology", ""]
+---
+
+## ¿Qué es?
+
+Seguridad en ingeniería del caos, o SCE en Inglés, es una disciplina basada en la [ingeniería del caos](/es/chaos-engineering/).
+SCE ejecuta experimentos de seguridad de manera proactiva en un sistema distribuido
+para aportar a la confianza del sistema en relación a su capacidad de soportar condiciones maliciosas o turbulentas.
```
Comment (ramrodo) [inline content/es/security-chaos-engineering.md:12]:
```suggestion
para aportar confianza en la capacidad del sistema de soportar condiciones maliciosas o turbulentas.
```

File: content/es/security-chaos-engineering.md [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+Los ingenieros a cargo de llevar a cabo esta práctica utilizan métodos científicos,
+incluyendo el estado inicial, hipótesis, verificación continua, lecciones aprendidas e implementación de mitigaciones.
+
+## Problema que aborda
+
+La mayor prioridad de los [ingenieros de site reliability](/es/site-reliability-engineering/) (o SRE en Inglés) e ingenieros de seguridad informática es
```
Comment (ramrodo) [inline content/es/security-chaos-engineering.md:18]: Podemos manejar el término como `ingenieros de confiabilidad del sitio` tal y como se manejó en [la localización del término SRE](https://github.com/cncf/glossary/blob/dev-es/content/es/site-reliability-engineering.md?plain=1#L12).

```suggestion
La mayor prioridad de los [ingenieros de confiabilidad del sitio](/es/site-reliability-engineering/) (o SRE en Inglés) e ingenieros de seguridad informática es
```

File: content/es/security-chaos-engineering.md [hunk 6]
```diff
@@ -0,0 +1,35 @@
...
+restaurar el servicio tan pronto como sea posible con el objetivo global de alcanzar cero interrupciones y minimizar el impacto al negocio.
+Tanto los SRE como los ingenieros de seguridad se enfrentan con incidentes y sus respectivas situaciones anteriores y posteriores.
+La mayoría de los problemas de seguridad son difíciles de encontrar y mitigar rápidamente, impactando la funcionalidad de la aplicación o el sistema.
+Adicionalmente, los incidentes de seguridad no son tan evidentes en la fase de desarrollo.
+
+## ¿Cómo ayuda?
+
+La seguridad en ingeniería del caos está construida alrededor de la [observabilidad](/es/observability/) y las prácticas de resiliencia informática.
```
Comment (ramrodo) [inline content/es/security-chaos-engineering.md:26]:
```suggestion
La Seguridad en Ingeniería del Caos está construida alrededor de la [observabilidad](/es/observability/) y las prácticas de resiliencia informática.
```

**Review by ramrodo — CHANGES_REQUESTED** [review]
Genial. Solo algunas sugerencias propuestas.

**Review by ramrodo — APPROVED** [review]
/lgtm

**Review by krol3 — APPROVED** [review]
lgtm

**Pull request merged**